OpenSim/Common/Component.cpp

```diff
@@ -615,86 +615,27 @@ void Component::setParent(const Component& parent)
 
 std::string Component::getFullPathName() const
```

Comment:
Considering that you use the term "absolute" in the Path classes, what about renaming this function to `getAbsolutePathName()`?